code/modules/atmos_automation/implementation/passive_gates.dm

```diff
@@ -0,0 +1,40 @@
+/datum/automation/set_passive_gate_state
+	name = "Passive Gate: Set Open/Closed"
+	var/gate=null
+	var/state=0
+
+	Export()
```

Comment:
I know all the other automations use it but I don't feel like we should be continuing to promote relative pathing

Reply:
@brndd you should be ashamed of yourself

Reply:
It wasn't removed from the other automations so I wanted to be consistent. I don't think it's all that bad here, this is almost a singleton datum and you should never see its procs called directly anywhere in the code.

Reply:
I'm gonna let it slide this time but this is a bad practice. We shouldn't be coding for consistency with oldcode if oldcode isn't up to modern standards. Hopefully someone down the line fixes all of them, yours included.